[2.7.2] #462 Add instructions for how to use PSACT (#489)
rancher/rancher-docs: docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/psa-config-template.md

Reviewer: Duplicated file and should be removed?

Author: That's odd -- I don't have that file anymore in my local, so I can't rm it using the command line. I guess I have to delete it through the web interface?

> ---

Author: Updated in [a980849](https://github.com/rancher/rancher-docs/pull/489/commits/a980849ee67f4f041f1f67944c5628a250506a5a)

## Assign a Pod Security Admissions (PSA) Configuration Template

You can assign a PSA template at the same time that you create a downstream cluster. You can also add a template by configuring an existing cluster.

### Assign a Template During Cluster Creation 

For RKE2 and K3s:

1. In the upper left corner, click **☰ > Cluster Management**.
1. On the **Clusters** page, click the **Create** button.
1. Select a provider.
1. On the **Cluster: Create** page, go to **Basics > Security**. 
1. In the **Default Pod Security Admission** dropdown menu, select the template you want to assign.

For RKE1:

1. In the upper left corner, click **☰ > Cluster Management**.
1. On the **Clusters** page, click the **Create** button.
1. Select a provider.
1. On the **Add Cluster** page, under **Cluster Options**, click **Advanced Options**.
1. In the **Pod Security Admission Configuration Template** dropdown menu, select the template you want to assign. 

### Assign a Template to an Existing Cluster

For RKE2 and K3s:

1. In the upper left corner, click **☰ > Cluster Management**.

> 1. Find the cluster you want to update in the **Clusters** table, and click the **⋮**.

Reviewer: ```suggestion
1. Find the cluster you want to update in the **Clusters** table, and click the **⋮**.
1. Select **Edit Config** .
```

1. Find the template you want to modify, and click the **⋮**.
1. Select **Edit Config** to edit the template.

### Allow Non-Admin Users to Manage PSA Templates

If you want to allow other users to manage templates, you can bind that user to a role that grants all verbs (`"*"`) on `management.cattle.io/podsecurityadmissionconfigurationtemplates`.

:::note warning
Any user that is bound to the above permission will be able to change the restriction levels on _all_ managed clusters which use a given PSA template, including ones that they have no permissions on.
:::

## Exempting Namespaces

If you assign the `rancher-restricted` template to a cluster, by default the restrictions are applied across the entire cluster at the namespace level. To exempt certain namespaces from this highly restricted policy, do the following: 

1. In the upper left corner, click **☰ > Cluster Management**.
1. Click **Advanced** to open the dropdown menu.
1. Select **Pod Security Admissions**.
1. Find the template you want to modify, and click the **⋮**.
1. Select **Edit Config**.
1. Click the **Namespaces** checkbox under **Exemptions** to edit the **Namespaces** field. 

> 

Reviewer: ```suggestion

:::note
You need to update the target cluster to make the new template take effect in that cluster. An update can be triggered by editing and saving the cluster without changing values. 
:::

```

1. In the **Default Pod Security Admission** dropdown menu, select the template you want to assign.

For RKE1:

1. In the upper left corner, click **☰ > Cluster Management**.
1. On the **Clusters** page, click the **Create** button.
1. Select a provider.
1. On the **Add Cluster** page, under **Cluster Options**, click **Advanced Options**.
1. In the **Pod Security Admission Configuration Template** dropdown menu, select the template you want to assign. 

### Assign a Template to an Existing Cluster

For RKE2 and K3s:

1. In the upper left corner, click **☰ > Cluster Management**.
1. Find the cluster you want to update in the **Clusters** table, and click the **⋮**.
1. Select **Edit Config** .
1. In the **Default Pod Security Admission** dropdown menu, select the template you want to assign.

For RKE1:

1. In the upper left corner, click **☰ > Cluster Management**.
1. Find the cluster you want to update in the **Clusters** table, and click the **⋮**.
1. On the **Add Cluster** page, go to **Cluster Options > Advanced Options**.
1. In the **Pod Security Admission Configuration Template**, select the template you want to assign. 

> ### Hardening the Cluster

Reviewer 2: ```suggestion
### Hardening an RKE2 Cluster
```

Reviewer 2: The above is a suggestion, because the CIS profile option is only applicable to RKE2 clusters. Otherwise we must let this clear in some other way.

> If you select the **rancher-restricted** template but don't select a **CIS Profile**, you won't meet required CIS benchmarks. See the [hardening guide](../../../pages-for-subheaders/rancher-security.md#rancher-hardening-guide) for more details.

Reviewer 2: ```suggestion
If you select the **rancher-restricted** template but don't select a **CIS Profile**, you won't meet required CIS benchmarks. See RKE2's [hardening guide](../../../pages-for-subheaders/rke2-hardening-guide.md) for more details.
```
